.github/workflows/create-release-with-version-submit.yaml

```diff
@@ -313,16 +313,27 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          RELEASE_ID=$(./scripts/create_draft_release.sh ${{ github.event.inputs.name }})
-          echo "release_id=$RELEASE_ID" >> $GITHUB_OUTPUT
+          gh release create ${{ inputs.name }} \
+            --draft \
+            --title "${{ inputs.name }}" \
+            --generate-notes \
+            --verify-tag
```

Comment:
Will this work? We create the tag after this step.

Reply:
Order changed in next push.